manifesto.html

```diff
@@ -0,0 +1,49 @@
+---
+# this is an empty front matter
+---
+{% include header.html lang="manifesto" %}
+  <main role="main" class="inner cover">
+    <h1 class="cover-heading">Bądźmy życzliwi ❤</h1>
+
+    <p class="lead">
+        W Hakierspejsie można spotkać wiele różnych osób. Różnice dotyczą nie
+        tylko naszych preferencji technologicznych ani poglądów w ogóle, ale
+        także tego jacy się urodziliśmy: na przykład naszej płci, stanu
+        zdrowia, orientacji seksualnej czy kraju, z którego pochodzimy.
+    </p>
+
+    <p class="lead">
+        Ta różnorodność sprawia, że dzięki różnym perspektywom i współpracy
+        potrafimy rozwiązać wiele problemów. Jednocześnie, jako, że
+        wiele ze wspomnianych cech nie jest naszym wyborem, a wręcz jest poza
+        naszą kontrolą, mogą one być czułym punktem, dlatego <em>bardzo ważne
+        jest żebyśmy się nawzajem szanowali.</em>
+    </p>
+
+    <p class="lead">
+        Hakierspejs jako organizacja nie zajmuje stanowiska na politycznym
+        spektrum i przynależą do nas osoby o różnych poglądach. Ten manifest
+        ma na celu nie indoktrynację, a stworzenie przestrzeni wolnej od
```

Comment:
Sugestia, która wynikła z #35 

```suggestion
        ma na celu stworzenie przestrzeni wolnej od
```